catt/controllers.py

```diff
@@ -221,17 +231,29 @@ def get_data(self, name):
 
 class CastStatusListener:
     def __init__(self, app_id, active_app_id):
-        self.app_id = app_id
         self.app_ready = threading.Event()
+        self.set_app_id(app_id, active_app_id)
```

Comment:
Please just add a ```backdrop_ready``` flag of the ```threading.Event``` kind, and then initiate it like ```app_ready```, and expand the logic in ```new_cast_status``` so the flag is set/cleared in the same manner as ```app_ready```.